div-calculator.rb

```diff
@@ -0,0 +1,75 @@
+# calculates answer after taking user input for operation to be 
+  
+# checks to make sure that symbol is valid
+def symbol_entry(symbol)
+  if symbol != "add" && symbol != "+" && symbol != "subtract" && symbol != "-" && symbol != "multiply" && symbol != "*" && symbol != "divide" && symbol != "/" && symbol != "^" && symbol != "%" && symbol != "exponent" && symbol != "module"
+    puts "Invalid symbol input!"
+  end 
+end
+ 
+# does the math according to numerical operator  
+def symbol_check(symbol, value1, value2) 
+print "Answer: " 
+  case symbol
+  when "add", "+"
+    print "#{value1} + #{value2} = " 
+    ans = value1 + value2
```

Comment:
You're doing both your calculations and your print statements in the same method! The further you get into the course, the more I'd encourage you to make sure your methods are doing just one "operation" (ie, just doing the math, just doing output, just sorting an array).